ui/src/table.tsx

```diff
@@ -117,7 +117,7 @@ interface TableGroup {
  * set to True, the form is not submitted automatically, and one or more buttons are required in the form to trigger
  * submission.
  * 
- * If `pagination` is set, you have to handle search/filter/sort/download/page_change/reset events yourself since
+ * If `pagination` is set, you have to handle search/filter/sort/download/page_change/reset/select events yourself since
```

Comment:
Is the imperative selection supported though?

Reply:
Good point, it is not.